app/views/admin/users/home.html.erb

```diff
@@ -11,7 +11,7 @@
                     <a class="btn-block btn-lg btn-info mt-4" href=/admin/users>Users</a>
                 </div>
                 <p class="mt-4">You can add or remove the administ-ration to users.</p>
-                <a class="index_list btn btn-primary mt-2" href="/users/1">Back</a>
+                <a class="index_list btn btn-outline-light mt-2" href="/users/1">Back</a>
```

Comment:
Please use link_to here not  anchor tag. it should be
  <%= link_to 'Back', admin_categories_path, class:"index_list btn btn-light mt-2" %>